src/components/scene/spectator.js

```diff
@@ -0,0 +1,40 @@
+var register = require('../../core/component').registerComponent;
+
+var bind = require('../../utils/').bind;
+var constants = require('../../constants/');
+var DEFAULT_SPECTATOR_CAMERA_ATTR = 'data-aframe-default-spectator-camera';
+
+/**
+ * spectator component.
+ * It injects a spectator camera if there's not one already defined.
+ */
+module.exports.Component = register('spectator', {
```

Comment:
Is a wrapper component necessary? I'd rather define my own camera via HTML, providing like id/position/rotation/scale with possibility of custom controls (maybe defaulting to orbit).

Reply:
You can provide your own camera too but it is nice to be able to do just:
`<a-scene spectator>`

Reply:
I think it's too much cruft, 40 LOC / documentation page / need to understand difference between `<a-scene spectator>` vs `<a-entity camera="spectator: true">`. Spectator camera will often be a deliberate feature addition, and in most cases we'd want to tailor the camera to the app which an injected camera won't help